### PR TITLE
Create clear_desk_clear_screen.md

### PR DESCRIPTION
**Pull request by EThom73**
A guide to our clear desk/clear screen policy to help our team ensure ongoing work is kept secure.

### CODE REVIEW

File: guides/office/clear_desk_clear_screen.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Clear Desk:
+- All ‘Confidential’ information must be hidden from view when not being actively used
+- Hard copies should be kept in locked drawers or returned to the appropriate place in the office 
+- Drawers/files containing sensitive information must be kept closed and locked. The keys to these drawers/files should be kept in a secure location and not left unattended at any time
+- Printouts should be immediately removed from the printer
+- When no longer needed, sensitive documents should be shredded in the office shredder
+- Passwords to confidential material should not be left on pieces of paper, sticky notes, or left unattended on a desk
+
+## Clear Screen:
+- Electronic copies of confidential materials should always be stored in the appropriate Made Tech tool (Google Team Drive, GitHub etc.)
```
Comment (chrisblackburn) [inline guides/office/clear_desk_clear_screen.md]: I think this is covered elsewhere. Unsure it is relevant to clear screen?

File: guides/office/clear_desk_clear_screen.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- If you need to leave your workstation for any reason, you should ensure your screen is locked and not visible
+- If you are using multiple screens, ensure you remain aware of what is displayed on both screens
+- When working outside the Made Tech Office (either from home, onsite with a client, or in a public space) ensure you remain aware of your surroundings and who may be able to view your screen. 
+
+### Devices:
+- When not in use, turning off your device to ensure disk encryption is enabled is important
```
Comment (chrisblackburn) [inline guides/office/clear_desk_clear_screen.md]: Turn over turning

File: guides/office/clear_desk_clear_screen.md (hunk 1 above)
Comment (EThom73) [reply guides/office/clear_desk_clear_screen.md]: removed 👍

File: guides/office/clear_desk_clear_screen.md (hunk 2 above)
Comment (EThom73) [reply guides/office/clear_desk_clear_screen.md]: changed 👍

File: guides/office/clear_desk_clear_screen.md [hunk 3]
```diff
@@ -11,13 +11,12 @@ Made Tech operates a Clear Desk/Clear Screen policy. This means that all sensiti
...
 
 ## Clear Screen:
-- Electronic copies of confidential materials should always be stored in the appropriate Made Tech tool (Google Team Drive, GitHub etc.)
 - If you need to leave your workstation for any reason, you should ensure your screen is locked and not visible
 - If you are using multiple screens, ensure you remain aware of what is displayed on both screens
 - When working outside the Made Tech Office (either from home, onsite with a client, or in a public space) ensure you remain aware of your surroundings and who may be able to view your screen. 
 
 ### Devices:
-- When not in use, turning off your device to ensure disk encryption is enabled is important
+- When not in use, turn off your device to ensure disk encryption is enabled is important
```
Comment (chrisblackburn) [inline guides/office/clear_desk_clear_screen.md]: I think this change has broken the sentence. Would make sense without the last couple of words now?

**Pull request merged**